mesh/mesh.cpp

```diff
@@ -4878,14 +4882,20 @@ int *Mesh::GeneratePartitioning(int nparts, int part_method)
    ElementToElementTable();
 
    partitioning = new int[NumOfElements];
-
    if (nparts == 1)
    {
       for (i = 0; i < NumOfElements; i++)
       {
          partitioning[i] = 0;
       }
    }
+   else if (nparts == NumOfElements)
+   {
+      for (i = 0; i < NumOfElements; i++)
+      {
+         partitioning[i] = i;
+      }
+   }
```

Comment:
This will be in conflict with https://github.com/mfem/mfem/pull/674/files#diff-7dc3e1a317e9f6a450c7cf9f0f1bc8e3

Reply:
I'll drop my commit

Reply:
I have dropped my commit